plugin/evm/uptime/pausable_manager_test.go

```diff
@@ -19,167 +19,195 @@ func TestPausableManager(t *testing.T) {
 	startTime := time.Now()
 	require := require.New(t)
 
-	// Case 1: Connect, pause, start tracking
-	{
-		up, clk, _ := setupTestEnv(nodeID0, startTime)
-
-		// Connect before tracking
-		require.NoError(up.Connect(nodeID0))
-		addTime(clk, time.Second)
-
-		// Pause before tracking
-		up.OnValidatorStatusUpdated(vID, nodeID0, false)
-
-		// Elapse Time
-		addTime(clk, time.Second)
-
-		// Start tracking
-		require.NoError(up.StartTracking([]ids.NodeID{nodeID0}))
-		currentTime := addTime(clk, time.Second)
-		// Uptime should not have increased since the node was paused
-		checkUptime(t, up, nodeID0, 0*time.Second, currentTime)
-
-		// Disconnect
-		require.NoError(up.Disconnect(nodeID0))
-		// Uptime should not have increased
-		checkUptime(t, up, nodeID0, 0*time.Second, currentTime)
+	tests := []struct {
+		name     string
+		testFunc func(t *testing.T, up PausableManager, clk *mockable.Clock, s uptime.State)
+	}{
+		{
+			name: "Case 1: Connect, pause, start tracking",
+			testFunc: func(t *testing.T, up PausableManager, clk *mockable.Clock, s uptime.State) {
+				// Connect before tracking
+				require.NoError(up.Connect(nodeID0))
```

Comment:
These require will capture the wrong `t` and panic, we can define the require per testFunc or use require.NoError(t, ...) etc.